### ISSUE TITLE
[Bug]: Field definition for field "key" not found.

### ISSUE THREAD

**Issue by DuckThom** [pimcore/data-importer#380]
### Expected behavior

System fields can be imported

### Actual behavior

System fields, such as 'key' cannot be imported as they don't have a field definition.

### Steps to reproduce

Pimcore: 11.1.1
Data importer: 1.8.5

![image](https://github.com/pimcore/data-importer/assets/6512901/09d06094-68bd-4985-b079-8f2d699df845)

1. Create a data import job
2. Map something to "SYSTEM Key"
3. Run import
4. Item cannot be imported due to the following error:

```
 ERR  11:03:33 ERROR     [DATA-IMPORTER] Error processing element: [data removed] Pimcore\Bundle\DataImporterBundle\Exception\InvalidConfigurationException: Field definition for field "key" not found. in /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Mapping/DataTarget/Direct.php:189
 ERR  Stack trace:
 ERR  #0 /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Mapping/DataTarget/Direct.php(150): Pimcore\Bundle\DataImporterBundle\Mapping\DataTarget\Direct->getFieldDefinition(Object(Pimcore\Model\DataObject\Product), 'key')
 ERR  #1 /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Mapping/DataTarget/Direct.php(80): Pimcore\Bundle\DataImporterBundle\Mapping\DataTarget\Direct->checkAssignData('Protein Iced Co...', Object(Pimcore\Model\DataObject\Product), 'getKey')
 ERR  #2 /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Processing/ImportProcessingService.php(224): Pimcore\Bundle\DataImporterBundle\Mapping\DataTarget\Direct->assignData(Object(Pimcore\Model\DataObject\Product), 'Protein Iced Co...')
 ERR  #3 /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Processing/ImportProcessingService.php(147): Pimcore\Bundle\DataImporterBundle\Processing\ImportProcessingService->processElement('products_genera...', Array, Object(Pimcore\Bundle\DataImporterBundle\Resolver\Resolver), Array)
 ERR  #4 /Volumes/Projects/[redacted]/pimcore/vendor/pimcore/data-importer/src/Command/ParallelProcessQueueCommand.php(70): Pimcore\Bundle\DataImporterBundle\Processing\ImportProcessingService->processQueueItem(16156841)
 ERR  #5 /Volumes/Projects/[redacted]/pimcore/vendor/webmozarts/console-parallelization/src/ParallelExecutor.php(251): Pimcore\Bundle\DataImporterBundle\Command\ParallelProcessQueueCommand->runSingleCommand('16156841', Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #6 /Volumes/Projects/[redacted]/pimcore/vendor/webmozarts/console-parallelization/src/ParallelExecutor.php(230): Webmozarts\Console\Parallelization\ParallelExecutor->runTolerantSingleCommand('16156841', Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Webmozarts\Console\Parallelization\Logger\StandardLogger))
 ERR  #7 /Volumes/Projects/[redacted]/pimcore/vendor/webmozarts/console-parallelization/src/ParallelExecutor.php(208): Webmozarts\Console\Parallelization\ParallelExecutor->processItems(Object(Webmozarts\Console\Parallelization\ChunkedItemsIterator), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Webmozarts\Console\Parallelization\Logger\StandardLogger), Object(Closure))
 ERR  #8 /Volumes/Projects/[redacted]/pimcore/vendor/webmozarts/console-parallelization/src/ParallelExecutor.php(87): Webmozarts\Console\Parallelization\ParallelExecutor->executeChildProcess(Object(Webmozarts\Console\Parallelization\Input\ParallelizationInput), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Webmozarts\Console\Parallelization\Logger\StandardLogger))
 ERR  #9 /Volumes/Projects/[redacted]/pimcore/vendor/webmozarts/console-parallelization/src/Parallelization.php(146): Webmozarts\Console\Parallelization\ParallelExecutor->execute(Object(Webmozarts\Console\Parallelization\Input\ParallelizationInput), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Webmozarts\Console\Parallelization\Logger\StandardLogger))
 ERR  #10 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/console/Command/Command.php(326): Pimcore\Bundle\DataImporterBundle\Command\ParallelizationAbstractCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #11 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/console/Application.php(1081): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #12 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/framework-bundle/Console/Application.php(91): Symfony\Component\Console\Application->doRunCommand(Object(Pimcore\Bundle\DataImporterBundle\Command\ParallelProcessQueueCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #13 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/console/Application.php(320): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand(Object(Pimcore\Bundle\DataImporterBundle\Command\ParallelProcessQueueCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #14 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/framework-bundle/Console/Application.php(80): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #15 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/console/Application.php(174): Symfony\Bundle\FrameworkBundle\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #16 /Volumes/Projects/[redacted]/pimcore/vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php(54): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
 ERR  #17 /Volumes/Projects/[redacted]/pimcore/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\ConsoleApplicationRunner->run()
 ERR  #18 /Volumes/Projects/[redacted]/pimcore/bin/console(20): require_once('/Volumes/Projec...')
 ERR  #19 {main}
```

**Comment by dvesh3** [comment]
fixed by #378

**Comment by DuckThom** [comment]
@dvesh3 Aah thanks! Should've seen that myself, forgot to check the open PRs!
